docs/src/pages/components/chips/ChipsArray.tsx

```diff
@@ -35,7 +35,7 @@ export default function ChipsArray() {
 
   const handleDelete = (chipToDelete: ChipData) => () => {
     if (chipToDelete.label === 'React') {
-      alert('Why would you want to delete React?! :)');
```

Comment:
This one might be the only one it the code base that is justified. Without, it's unclear why clicking the delete button does nothing for React while it removes the other tags.

Reply:
I'll rewrite this so it isn't handled in the first place. In the end why mark a chip as deletable if it isn't?

Reply:
Right 👌